components/eam/bld/configure

```diff
@@ -2538,10 +2539,14 @@ sub write_filepath
 
           # advection scheme
           my $crm_adv = $cfg_ref->get('crm_adv');
-          if ($crm_adv eq 'UM5') {
-              print $fh "$camsrcdir/eam/src/physics/crm/sam/ADV_UM5\n";
+          if (defined $opts{'crm_adv'}) {
```

Comment:
is 'if defined crm_adv' statement necessary here? for all crm_adv (including the case where crm_adv is not defined by a user or a compset) other than 'UM5' have the same $fh which is ADV_MPDATA directory.

Reply:
We need to check that it is defined because I changed the default to have this undefined, but I guess I can write this in a more sensible way, so I'll do that.

Reply:
done.

Reply:
looks good.